src/coveralls.jl

```diff
@@ -94,10 +94,7 @@ module Coveralls
             println("Submitting data to Coveralls...")
         end
 
-        url = "https://coveralls.io/api/v1/jobs"
-        body = HTTP.Form(makebody(data))
-        headers = ["Content-Type" => "multipart/form-data; boundary=$(body.boundary)"]
-        req = HTTP.post(url, headers, body)
+        req = HTTP.post("https://coveralls.io/api/v1/jobs", HTTP.Form(makebody(data)))
```

Comment:
missing an identical update later in the file

Reply:
Good catch, thanks!